ios/brave-ios/Sources/Playlist/PlaylistDownloadManager.swift

```diff
@@ -715,7 +715,7 @@ private class PlaylistFileDownloadManager: NSObject, URLSessionDownloadDelegate
       // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range
       request.addValue("bytes=0-", forHTTPHeaderField: "Range")
       request.addValue(UUID().uuidString, forHTTPHeaderField: "X-Playback-Session-Id")
-      request.addValue(UserAgent.userAgentForIdiom(), forHTTPHeaderField: "User-Agent")
+      request.addValue(UserAgent.mobile, forHTTPHeaderField: "User-Agent")
```

Comment:
Will this have any implications with Playlist on iPad? cc @Brandon-T

Reply:
None. Kyle DM'd be and I tested them all :) so I approved :D.
Good looking out and thanks for checking :)